### ISSUE TITLE
Improve development environment setup documentation

### ISSUE THREAD

**Issue by noquierouser** [lyzs90/vuewp#6]
When attempting to setup a development environment, I followed the steps in the Development section in order, which isn't in the actual order to setup the environment. In particular, steps related to the `.env` file and running `docker-composer up` should be better clarified for newcomers.

The proposed order should be:
- First, copy `.env.example` to `.env` and change the variables in it.
- Then run `docker-composer up -d`.

Also, in a fresh install of docker, and after running `docker-composer up -d`, `docker-machine` reports no running machines despite `docker ls` showing both `wordpress:latest` and `mysql:5.7` containers running.

Personally I think this is a fantastic idea to setup a Wordpress development environment, mostly for newcomers to technologies like Docker and Vue, but a good documentation is crucial to understand what is going on the insides of the project.

**Comment by lyzs90** [comment]
@noquierouser certainly, moving env var setup before `docker-compose up` is a valid point

Perhaps we can also remove the part on using docker machine to get container ip since it doesn't work for everyone. For me, I installed docker toolbox which came bundled with docker machine. But I imagine everyone would have a different setup

Do u mind creating a simple PR with the changes? Thanks for contributing!
